web/tests/Web.E2ETests/Steps/School/CompareYourCostsSteps.cs

```diff
@@ -130,6 +130,20 @@ public async Task ThenTheSectionIsHidden(string chartName)
         await _comparisonPage.IsSectionVisible(ChartNameFromFriendlyName(chartName), false, "Show", "chart");
     }
 
+    [When("I click on how we choose similar schools")]
+    public async Task ClickComparatorSetDetails()
```

Comment:
This should be `WhenIClickOnHowWeChooseSimilarSchools` to follow SpecFlow naming convention for step definitions

Reply:
ahh yes good catch. fixed it now.